olxcleaner/loader/xml_exceptions.py

```diff
@@ -87,6 +87,14 @@ def __init__(self, filename, **kwargs):
         super().__init__(filename)
         self._description = f"The tag {kwargs['edxobj']} tag appears to be pointing to itself"
 
+class PointerAlreadyPointedAt(CourseError):
+    """A tag appears to be pointing to itself."""
+    _level = ErrorLevel.ERROR
+
+    def __init__(self, filename, **kwargs):
+        super().__init__(filename)
+        self._description = f"The {kwargs['edxobj']} tag has been pointed to and is unexpectedly empty."
```

Comment:
This reads like it should be an UnexpectedlyEmpty error. I know these are a bit nitpicky, but the whole point of this project is to make human-readable error messages, so it's important to get these right!